app.js

```diff
@@ -43,13 +43,13 @@ const flash = require('connect-flash');
 var logger = require('./server/logger.js');
 const path = require('path');
 
-// set up rate limiter: maximum of 120 requests per 10 seconds
+// set up rate limiter: maximum of 500 requests per 10 seconds
 var rateLimit = require('express-rate-limit');
 var limiter = rateLimit({
   windowMs: 1*10*1000, // 10 seconds
-  max: 120
```

Comment:
Honestly, limit of 120 requests per client in a 10 second window seems reasonable to me. @CallumRobb-ScottLogic do you think of any use case why we should support more requests?

Reply:
So we checked and each page will give 20+ requests on a single refresh if we are doing a longer test with an edited page this may become 40+ requests and if we are switching pages and going back to check status of edited page that only leave us 3 refreshes